### PR TITLE
Remove workaround for not being able to override generic method

### PR DESCRIPTION
**Pull request by Nirma**
Remove workaround for not being able to override a generic method in a generic subclass.
Moved the implementation of the methods found in `_ContiguousArrayStorage1` directly over to `_ContiguousArrayStorage<Element>` and removed `_ContiguousArrayStorage1`.

@gribozavr Thanks for pointing out this spot! How does it look?

### CODE REVIEW

**Comment by gribozavr** [comment]
@swift-ci Please test

**Comment by gribozavr** [comment]
@Nirma The patch looks good.  Let's see what the CI says.

**Comment by Nirma** [comment]
@gribozavr Sorry for that I pushed the fix, would you mind letting CI go again?

**Comment by gribozavr** [comment]
@swift-ci Please test

**Comment by gribozavr** [comment]
@Nirma OS X CI finished, could you take a look at the issues?

**Comment by Nirma** [comment]
@gribozavr Thanks for kicking off the tests again, the previous override error is gone but the failure this time appears to be:
`/home/buildnode/jenkins/workspace/swift-PR-Linux/swiftpm/Tests/Build/DescribeTests.swift:14:13: error: type 'DescribeTests' does not conform to protocol 'XCTestCaseProvider'`

Is this related to this change?

**Comment by gribozavr** [comment]
@Nirma No, swiftpm failure is unrelated (exists on master currently).

**Comment by gribozavr** [comment]
@swift-ci Please test

**Comment by gribozavr** [comment]
@Nirma Looks like there's a real OS X failure this time.

**Comment by Nirma** [comment]
@gribozavr The generic syntax was off, sorry about that. How does it look now?

**Comment by gribozavr** [comment]
@swift-ci Please test

File: stdlib/public/core/ContiguousArrayBuffer.swift [hunk 1]
```diff
@@ -107,6 +76,19 @@ final class _ContiguousArrayStorage<Element> : _ContiguousArrayStorage1 {
...
+    var result: Element? = nil
+    try self._withVerbatimBridgedUnsafeBufferImpl {
+      result = try body($0)
+    }
+    return result
+  }
+    
   /// If `Element` is bridged verbatim, invoke `body` on an
   /// `UnsafeBufferPointer` to the elements.
   internal final override func _withVerbatimBridgedUnsafeBufferImpl(
```
Comment (gribozavr) [inline stdlib/public/core/ContiguousArrayBuffer.swift]: Does this function still override anything?  Do we need to factor it as a separate function, or can we inline it into its only (as far as I can see) caller -- `_withVerbatimBridgedUnsafeBuffer `?

**Comment by Nirma** [comment]
@gribozavr No it does not, so rather than just removing the override I can write it inline!

**Comment by gribozavr** [comment]
@swift-ci Please test

**Comment by gribozavr** [comment]
@Nirma Seems like we have test failures on OS X.

**Comment by DougGregor** [comment]
There are non-trivial failures to resolve here and the PR hasn't been updated in 20 days; closing out the stale PR.

**Pull request closed without merging**